docs/advanced/subscriptions.md

```diff
@@ -29,21 +29,30 @@ A complete example can be found [in `SubscriptionDatafetcher.java`](https://gith
 
 The GraphQL specification doesn't specify a transport protocol.
 WebSockets are the most popular transport protocol however, and are supported by the DGS Framework.
-Apollo defines a [sub-protocol](https://github.com/apollographql/subscriptions-transport-ws/blob/master/PROTOCOL.md), which is supported by client libraries and implemented by the DGS framework.
+Apollo defines a [sub-protocol](https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md), which is supported by client libraries and implemented by the DGS framework.
```

Comment:
This is not yet available

Reply:
Not sure what you mean here, I thought DGS supported websockets?